### DISCUSSION TITLE
Supports Chrome extension format or Chrome OS compatible

### DISCUSSION THREAD

**Discussion by SoonieChoi** [dbeaver/dbeaver#32348] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

I am using Chrome OS and cannot install dbeaver.

**Describe the solution you'd like**

Please release the dbeaver app or extension on the Google Store/Google Web Store.

**Comment by E1izabeth** [comment, marked as answer]
DBeaver is based on Eclipse IDE, so the question is more like "Is it possible to install Eclipse on Chrome OS?". And the answer is that it's not possible. Here is the discussion on stackoverflow about that https://stackoverflow.com/questions/11418183/eclipse-for-chrome .
Also I've found the workaround about enabling Linux environment under Chrome OS and installing dbeaver with flatpak `flatpak install flathub io.dbeaver.DBeaverCommunity`. Maybe you can try this.
![Screenshot-2022-10-12-4 57 33-AM-e1665565083256](https://github.com/dbeaver/dbeaver/assets/28875055/1215c2ee-eb07-4ed7-b1db-451466650f82)

**Comment by E1izabeth** [comment]
@SoonieChoi 
Also I would recommend you to take a look at our web product CloudBeaver. You can try it here https://demo.cloudbeaver.io/ to decide if it fits your needs
Here is the link to CloudBeaver repository https://github.com/dbeaver/cloudbeaver
Here is the documentation about CloudBeaver  https://dbeaver.com/docs/cloudbeaver/
